dockers/docker-dhcp-relay/docker-dhcp-relay.supervisord.conf.j2

```diff
@@ -43,8 +43,16 @@ isc-dhcp-relay-{{ vlan_name }}
 
 
 {# Create a program entry for each DHCP relay agent instance #}
+{% set relay_for_ipv4 = { 'flag': False } %}
 {% for vlan_name in VLAN %}
 {% if VLAN[vlan_name]['dhcp_servers'] %}
+{% for dhcp_server in VLAN[vlan_name]['dhcp_servers'] %}
+{% if  dhcp_server | ipv4 %}
```

Comment:
Remove extra space after "if"

Reply:
OK, I will change it.Thanks for your comment.